packages/terra-props-table/tests/jest/generatePropType.test.js

```diff
@@ -1,23 +1,25 @@
 const generatePropType = require('../../bin/generateMarkdown/generatePropType');
 
-
 describe('generatePropType', () => {
-  it('should return node string', () => {
-    expect(generatePropType({ name: 'node' })).toEqual('`node`');
+  it('should return "node" for node', () => {
+    const generatedPropType = generatePropType({ name: 'node' });
+    expect(generatedPropType).toEqual('`node`');
+    expect(generatedPropType).toMatchSnapshot();
   });
 
-
-  it('should return object string', () => {
-    expect(generatePropType({ name: 'object' })).toEqual('`object`');
+  it('should return "object" for object', () => {
+    const generatedPropType = generatePropType({ name: 'object' });
+    expect(generatedPropType).toEqual('`object`');
+    expect(generatedPropType).toMatchSnapshot();
```

Comment:
Since you are explicitly testing a function call, I would go with the toEqual tests that you have